### DISCUSSION TITLE
how to work with custom command  cd ?

### DISCUSSION THREAD

**Discussion by WingDust** [nushell/nushell#10919] [Q&A]
```sh
def site-packages [] {
    cd (python3 -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")
}

```
function is right, but run site-packages no do cd, 

manual run  `cd (python3 -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")` is work fine

**Comment by amtoine** [comment]
you need to use `def --env` to allow the `cd` command to mutate the environment and thus move you to the target directory :wink: 
```nushell
def --env site-packages [] {
    cd (python3 -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")
}
```
